tests/test_ssl.py

```diff
@@ -347,6 +347,44 @@ def test_SSLeay_version(self):
         self.assertEqual(len(versions), 5)
 
 
+@pytest.fixture
+def context():
+    """
+    A simple TLS 1.0 context.
+    """
+    return Context(TLSv1_METHOD)
+
+
+class TestContext(object):
+    @pytest.mark.parametrize("cipher_string", [
+        b"hello world:AES128-SHA",
+        u"hello world:AES128-SHA",
+    ])
+    def test_set_cipher_list(self, context, cipher_string):
+        """
+        :meth:`Context.set_cipher_list` accepts both :py:obj:`bytes` naming the
```

Comment:
This says "both bytes", which implies that the next bit would be "and unicode strings`. Did I misunderstand this?